tests/cupy_tests/manipulation_tests/test_kind.py

```diff
@@ -7,3 +7,15 @@
 class TestKind(unittest.TestCase):
 
     _multiprocess_can_split_ = True
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal(type_check=False)
+    def test_asfortranarray1(self, xp, dtype):
+        x = xp.zeros(2, 3)
```

Comment:
It should be `xp.zeros((2, 3))`. So this caused an error, but it is caught and grasped [here](https://github.com/pfnet/chainer/blob/master/cupy/testing/helper.py#L24).